### ISSUE TITLE
Use actual links within navigations for the current page

### ISSUE THREAD

**Issue by lizardat** [contao/contao#9589]
### Affected version(s)

5.7.1

### Description

the last entry in breadcrumb is marked in mod_breadcrumb as <li .... aria-current="page">.... This current page should be with a link as all other entries, as aria features are supposed to be used on interactive elements only.

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current

**Comment by fritzmg** [comment]
This means we would need to change it in `nav_default` too though. Not sure if we should do this in a bugfix release?

**Comment by zoglo** [comment]
I agree with Fritz here.
As already mentioned, I would remove `aria-current` as the bugfix here.

If we make it a link, it will `visually` look like every other link in the breadcrumb, forcing people to **_change the visuals_**_** in a **bugfix** version.

Please read the following (from https://www.w3.org/TR/wai-aria-1.2/#aria-current):

> The [aria-current](https://www.w3.org/TR/wai-aria-1.2/#aria-current) attribute is used when an element within a set of related > elements <ins>**_is visually styled_**</ins> to indicate it is the current item in the set. For example:
> 
> - A page token used to indicate a link within a set of pagination links, where the link is <ins>**_visually_**</ins> styled to represent the currently-displayed page.
> - A step token used to indicate a link within a step indicator for a step-based process, where the link is <ins>**_visually_**</ins> styled to represent the current step.
> - A location token used to indicate the image that is <ins>**_visually_**</ins> highlighted as the current component of a flow chart.
> - A date token used to indicate the current date within a calendar.
> - A time token used to indicate the current time within a timetable.

Option 2 would be perfect as a feature, coincidentally also being in the next version, a major version (where people need more attention as there may be BC breaks, albeit a small one like this, one that would however be expected)

**Comment by zoglo** [comment]
> I consider links to the official documentation of aria more helpful than expressing a personal opinion.
> 
> But you are right: removing aria-current from the non-linked li is better than keeping it.

I am not talking about simply sending links, you also have to say what paragraph you mean. I also messaged an accessibility expert, hoping to get an answer and will see if they'll have some more insights for us as

**Comment by zoglo** [comment]
I'd be for Contao 6 due to the nature of the breaking change 👍
